models/pp_cond_exp_mc_urbanczik.h

```diff
@@ -107,82 +107,140 @@ class pp_cond_exp_mc_urbanczik_parameters
+   neurons it is interpeted as the dendritic delay. b) Two two-compartment
+   neuron models composed of a somatic (green) and a dendritic (blue)
+   compartment. The soma of the presynaptic neuron is connected to the dendrite
+   of the postsynaptic neuron. The synaptic delay is located behind the synapse
+   and before the dendrite. c) Time trace of the State variables that enter the
+   Urbanczik-Senn rule. Due to the synaptic delay :math:`d`, the presynaptic
+   spike train (top) is combined with a delayed version of the postsynaptic
+   quantities; the dendritic membrane potential (middle) and the somatic spike
+   train (bottom).
+
+
+Parameters
+++++++++++
 
 The following parameters can be set in the status dictionary. Parameters
 for each compartment are collected in a sub-dictionary; these sub-dictionaries
 are called "soma" and "dendritic", respectively. In the list below,
 these parameters are marked with an asterisk.
 
-\verbatim embed:rst
-
-============ ======= ==========================================================
- V_m*        mV      Membrane potential
- E_L*        mV      Leak reversal potential
- C_m*        pF      Capacity of the membrane
- E_ex*       mV      Excitatory reversal potential
- E_in*       mV      Inhibitory reversal potential
- g_L*        nS      Leak conductance
- tau_syn_ex* ms      Rise time of the excitatory synaptic alpha function
- tau_syn_in* ms      Rise time of the inhibitory synaptic alpha function
- I_e*        pA      Constant input current
- g_sp        nS      Coupling between soma and dendrite
- g_ps        nS      Coupling between dendrite and soma
- t_ref       ms      Duration of refractory period
-============ ======= ==========================================================
-\endverbatim
-
-Example:
-See pynest/examples/urbanczik_synapse_example.py.
+============   =====   =====================================================
+ V_m*           mV      Membrane potential
+ E_L*           mV      Leak reversal potential
+ C_m*           pF      Capacity of the membrane
+ E_ex*          mV      Excitatory reversal potential
+ E_in*          mV      Inhibitory reversal potential
+ g_L*           nS      Leak conductance
+ tau_syn_ex*    ms      Rise time of the excitatory synaptic alpha function
+ tau_syn_in*    ms      Rise time of the inhibitory synaptic alpha function
+ I_e*           pA      Constant input current
+ g_sp           nS      Coupling between soma and dendrite
+ g_ps           nS      Coupling between dendrite and soma
+ t_ref          ms      Duration of refractory period
+============   =====   =====================================================
+
+See :doc:`../auto_examples/urbanczik_synapse_example` to learn more.
 
 Remarks:
 
-The neuron model uses standard units instead of the unitless quantities used in [1].
+The neuron model uses standard units instead of the unitless quantities used in
```

Comment:
What are standard units?

Reply:
@jessica-mitchell With the term "standard units" we refer to the units that are used in NEST in all models, e.g. mV, nS, ms, pA, ...

Reply:
```suggestion
The neuron model uses standard units of NEST instead of the unitless quantities used in
```

Reply:
@jessica-mitchell Yes, I agree.

Reply:
> I'd suggest to explicitly state that

This suggestion still needs to be addressed.